@here/harp-mapview/lib/geometry/TileGeometryLoader.ts

```diff
@@ -262,6 +262,32 @@ export class TileGeometryLoader {
         enabledKinds: GeometryKindSet | undefined,
         disabledKinds: GeometryKindSet | undefined
     ) {
+        const decodedTile = this.m_decodedTile;
+
+        // Just a sanity check that satisfies compiler check below.
+        if (decodedTile === undefined) {
+            this.finish();
+            return;
+        }
+
+        this.tile.mapView.taskQueue.add({
+            execute: this.prepare.bind(this, enabledKinds, disabledKinds),
+            group: TileTaskGroups.CREATE,
+            getPrio: this.getPriority.bind(this),
```

Comment:
Will the priority change? If it changes, will `TaskQueue` handle it properly?
Maybe for simplicity just call `this.getPriority` here and assign it to the task?

Reply:
Yes, it will change, as the Camera moves, the priority will change based on the distance of the tile to the camera. 
I added a test to showcase the dynamic update of a Tasks priority